### PR TITLE
Add the Kubernetes 1.20 jobs

### LINKED ISSUE #1218: Create Kubernetes 1.20 presubmits and periodics for E2E tests

### PR DESCRIPTION
**Pull request by xmudrii**
**What this PR does / why we need it**:

Considering that the API server issue will be fixed in #1243, We can now add the Kubernetes 1.20 jobs.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1218

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/assign @kron4eg

### CODE REVIEW

**Comment by xmudrii** [comment]
/hold
until #1243 is not merged

**Review by kron4eg — APPROVED** [review]
/hold cancel

**Pull request merged**
